extensions/grpc/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -8,4 +8,8 @@ metadata:
     - "web"
     - "serialization"
     - "reactive"
-  status: "experimental"
\ No newline at end of file
+  status: "experimental"
+  codestart:
+    name: "grpc"
+    languages:
+      - "java"
```

Comment:
you need to add the  artifact: "io.quarkus:quarkus-project-core-extension-codestarts"